solidity/contracts/system/TBTCSystem.sol

```diff
@@ -48,9 +48,9 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
     uint256 constant pausedDuration = 10 days;
 
     TBTCDepositToken tbtcDepositToken;
-    address public keepVendor;
-    address public priceFeed;
-    address public relay;
+    IBTCETHPriceFeed priceFeed;
+    IBondedECDSAKeepVendor keepVendor;
+    IRelay relay;
```

Comment:
These were declared `public` before, we should probably keep that explicit declaration? Or does leaving off `public` do something else? (Sorry, occasional Solidity gaps exist.)

Reply:
> Or does leaving off public do something else

Nope, it's just public by default. I'll add the visibility back for consistency.